### PR TITLE
adding in ability to generate the text for a tweet to prompt everyone for availability

### PR DESCRIPTION
**Pull request by alterisian**
… for their availability. Since we do not need to geocode for this call, I have introduce a flag to turn geocoding off. It maybe useful to also change VCR from run once in the future too. As we have now many people in the mob, the string is longer than a tweet, so I have introduced split_string which breaks up the string at the end of the word before 280 characters and the tweet size limit. New tests introduced for this new functionality also. ChatGPT helped with split_string

### CODE REVIEW

**Comment by alterisian** [comment]
Corresponds with issue #34

File: helloworld.rb [hunk 1]
```diff
@@ -89,6 +96,27 @@ def generate_tweet(people_west_of, location)
...
+      return [str]
+    else
+      first_part = str[0..279]
+      last_space_index = first_part.rindex(/\s/)
+      if last_space_index.nil? || last_space_index == 0
+        # no space found or space is at the beginning of the string
+        return [first_part, str[280..-1]]
+      else
+        # split the string at the last space before 280 characters
+        return [str[0..last_space_index-1], str[last_space_index+1..-1]]
```
Comment (Animesh-Ghosh) [inline helloworld.rb:110]: Might be good if we get rid of the magic numbers and use the `TWEET_CHARACTER_LIMIT` constant that's already defined. I guess this part was generated via ChatGPT?

**Review by Animesh-Ghosh — APPROVED** [review]
Just added one comment but the rest looks good!

File: helloworld.rb (hunk 1 above)
Comment (alterisian) [reply helloworld.rb:110]: Great spot, and totally need to refine my ChatGPT prompts to include the constants. I was developing a project that would have taken the existing code as an argument, but ran out of API credits. So this was via the web chat. Good to figure how we get the machine working for us. 

Updating accordingly.

**Pull request merged**